static/js/chart/draw_choropleth.ts

```diff
@@ -152,6 +152,20 @@ const onMouseMove = (
     .style("top", d3.event.offsetY + topOffset + "px");
 };
 
+const onMapClick = (domContainerId: string) => (
+  geo: { properties: { geoDcid: string } },
+  index
+) => {
+  window.open(REDIRECT_BASE_URL + geo.properties.geoDcid, "_blank");
```

Comment:
nit: In a follow up PR, it would be even better to carry out the topic from the url, which we don't do in bar chart url either.

Reply:
should I try to make the bar chart url also carry out the topic from the url then?

Reply:
Yep!